spring-boot/admin/src/main/resources/templates/admin/fragments/main/dashboard.html

```diff
@@ -1,13 +1,13 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org"
-      xmlns:sec="http://www.thymeleaf.org/thymeleaf-extras-springsecurity4">
```

Comment:
Was this removed on purpose? Does it have a relation to this merge request?

Reply:
This was removed from Intellij, as it seems that is not used (it appears in grey), but as was there before, I added it back

Reply:
So consider it Done